Create legend_belly_dancer_encounter.nut

Chilly-Maher opened the pull request:
Encounter-ifed belly dancer recruit. Notable changes are cutting the local town_name and using settlement instead, as well as cutting what I am hoping are now redundant event checks

Chilly-Maher committed:
Create legend_belly_dancer_encounter.nut

Encounter-ifed the belly dancer event, notable changes are not using local town_name and using settlement, as well as cutting some ~~hopefully~~ redundant event checks in isValid

TheCutestSkinghoul commented on scripts/encounters/events/legend_belly_dancer_encounter.nut:
```diff
@@ -0,0 +1,225 @@
+    function createScreens() {
+        this.m.Screens.extend({
+			ID = "A",
+			Text = "[img]gfx/ui/events/event_163.png[/img]{A belly dancer magnetizes %settlement%\'s center plaza. Rhythmic movements on their own can coerce a beggar to donating a crown, but with the stage of the whole plaza it is enough to draw crowds and with it heaps of gold. Masked by green silk, nearly see through, and clothed in thin silks with the whole midriff exposed, the dancer is no doubt an e ...
```
'Donating' is perhaps a misleading word here. Being maudlin for a moment, better connotations such as 'sacrifice' may be more on-brand for this universe? Let me know what you think.

Permanence: you don't need to state 'this' or 'that' when your subject is singular and presented recently as a result of said action taking place.
> The crowd claps gently to this. -> The crowd claps gently.

TheCutestSkinghoul commented on scripts/encounters/events/legend_belly_dancer_encounter.nut:
```diff
@@ -0,0 +1,225 @@
+		});
+		this.m.Screens.push({
+			ID = "C",
+			Text = "[img]gfx/ui/events/event_163.png[/img]{After the belly dancer takes your crown you wait around for the show to end. You approach as she picks up herthings. She looks at you with a wry smile.%SPEECH_ON%Ah, an admirer. Sorry, only one show tonight, good stranger.%SPEECH_OFF%You shake your head and ask if she knows anything about fighting. She nods.%SPEECH_ON%Of course I do. The Gilded On ...
```
**A few suggestions here:**
> After the belly dancer takes your crown you wait around for the show to end. You approach as she picks up herthings. She looks at you with a wry smile.%SPEECH_ON%Ah, an admirer. Sorry, only one show tonight, good stranger.%SPEECH_OFF%
- herthing -> her things <- I also think there should be specific item(s) here rather than 'things', have a think about this and what she may carry.
- 'good stranger' seems slightly out of place here.

> You shake your head and ask if she knows anything about fighting. She nods.%SPEECH_ON%Of course I do. The Gilded One\'s gleam is upon us all, but not at all hours or days. Sometimes we must find our own way through the dark. I take it by your dress that you are a Crownling, putting that blade of yours where it does and sometimes does not belong.%SPEECH_OFF%
- I think 'Of course I do' needs to heave some emphasis, such as 'Of course I do!' or 'Of course!' <- or perhaps remove this bit altogether and just start at 'The [...]' and let the reader draw conclusions.
- Sometimes we must find our own way through the dark. -> Sometimes, we must find our own way through the dark.

> You nod and ask her if she\'d be interested in joining. She goes bowlegged and sinks to the ground like a collapsing truss. She counts her crowns.%SPEECH_ON%I\'m not sure if you have a good eye for the wandering nature of ones such as myself. Perhaps you saw a vocational tiredness not even I was aware of until this present moment. That said, you\'ll have to try harder to get me to go around killing for coin.%SPEECH_OFF%}";
- I think a better verb than 'goes' would be good here. Eg, she drops, she slinks, etc.
- Perhaps you saw a vocational tiredness not even I was aware of until this present moment. <- in my option this sentence doesn't really fit with the narrative.
- I recommend using a trail off when a matter is not resolved at the end of a character talking: '[...] try harder to get me to go around killing for coin.' ->  '[...] try harder to get me to go around killing for coin...'

TheCutestSkinghoul commented on scripts/encounters/events/legend_belly_dancer_encounter.nut:
```diff
@@ -0,0 +1,225 @@
+    })
+		this.m.Screens.push({
+			ID = "D",
+			Text = "[img]gfx/ui/events/event_163.png[/img]{You assuage her ego by saying she\'s one of the best with the blade you\'ve seen. The dancer turns her hands to the dust, her fingers sliding beneath each coin and flipping it into her clay pot. Her left hand reaches across the ground, but as this catches your eye, her right hand snatches a blade that had been entirely buried beneath the sands. Sh ...
```
Maybe add a synonym to replace 'buried', like 'concealed'?

Chilly-Maher replied:
Missed this one tbh, but I think buried works here

TheCutestSkinghoul reviewed (changes requested):
Very good, just one spelling error and a few suggested refinements, please review and let me know what you think.

Chilly-Maher committed:
Update legend_belly_dancer_encounter.nut

Edited everywhere Luft saw holes, took some creative liberties here and there, lmk how it is
